include/ibamr/BrinkmanAdvDiffSemiImplicitHierarchyIntegrator.h

```diff
@@ -150,6 +150,23 @@ class BrinkmanAdvDiffSemiImplicitHierarchyIntegrator : public AdvDiffSemiImplici
         return d_brinkman_penalization;
     } // getBrinkmanPenalization
 
+    /*!
+     * \brief Set the boolean to be TRUE for the time independent Brinkman problems.
+     * Default option is false.
+     */
+    void setBrinkmanTimeIndepedent(SAMRAI::tbox::Pointer<SAMRAI::pdat::CellVariable<NDIM, double> > Q_var,
```

Comment:
suggest renaming it to ` setBrinkmanVariableTimeIndepedent`

Reply:
I think we settled on `setTransportQuantityTimeIndependent()`: hence this is resolved.